deploy-agent/deployd/common/stats.py

```diff
@@ -27,10 +28,20 @@ def __exit__(self, stats, sample_rate=1.0, tags=None):
         pass
 
 
+def _add_default_tags(tags=None):
+    if not __version__:
```

Comment:
maybe a comment `# defensive case, should not be hit`

Reply:
added!